armory/__main__.py

```diff
@@ -108,6 +117,9 @@ def run(command_args, prog, description):
         default=False,
```

Comment:
`dest` can be removed and `action`, `const`, and `default` collapsed to `action="store_true"`